CHANGELOG.rst

```diff
@@ -4,6 +4,13 @@ Purestorage.Fusion Release Notes
 
 .. contents:: Topics
 
+unreleased
```

Comment:
Do not change this file. It is automatically updated when the Collection is created and released